test/views/html/ContributionsEarningsSpec.scala

```diff
@@ -22,7 +22,8 @@ import play.twirl.api.Html
 import utils.GmpViewSpec
 
 class ContributionsEarningsSpec extends GmpViewSpec {
-  override def view: Html = views.html.contributions_earnings(calculationResponse)
+  lazy val gmpMain = app.injector.instanceOf[gmp_main]
+  override def view: Html = new views.html.contributions_earnings(gmpMain)(calculationResponse)
```

Comment:
This could be retrieved through the Views class (doesn't really matter because it's a test)